qdrant-landing/content/blog/qdrant-relari.md

```diff
@@ -0,0 +1,289 @@
+
+![Top K](/blog/qdrant-relari/topk.png)
+
+We can even look at individual cases in the UI to get more insight into why our retriever failed.
+
+![Top K UI](/blog/qdrant-relari/topk2.png)
+
+Relari and Qdrant can also be combined to evaluate your hybrid search. With this information, you can further tweak the parameter or try a different RAG architecture to select the best-performing strategies for your dataset.
+
+## Auto Prompt Optimization
+
+In conversational applications like chatbots, **Auto Prompt Optimization (APO)** improves the app’s ability to communicate. This technique adapts the chatbot's responses over time, learning from past interactions to adjust phrasing for better results.
+
+For example, in a customer service chatbot, the phrasing of questions can significantly affect user satisfaction. Consider ordering in French versus English in a Parisian cafe—the same information may be transmitted, but it will be received differently.
+
+![French Cafe](/blog/qdrant-relari/cafe.jpg)
+
+Auto Prompt Optimization continuously refines the chatbot’s responses to improve user interactions. Here's how you can implement APO with Relari:
+
+### Set up base prompt
+
+With Auto Prompt Optimization, you can define a system prompt and inspect the results at every iteration of the interaction.
+
+```python
+from relari.core.types import Prompt, UserPrompt
+
+base_prompt = Prompt(
+    system="You are a GitLab legal policy Q&A bot. Answer the following question given the context.",
+    user=UserPrompt(
+        prompt="Question: $question\n\nContext:\n$ground_truth_context",
+        description="Question and context to answer the question.",
+    ),
+)
+```
+
+#### Set up task id
+
+```python
+task_id = client.prompts.optimize(
+    name="GitLab Legal Policy RAG Prompt",
+    project_id=proj["id"],
+    dataset_id=dataset_info["id"],
+    prompt=base_prompt,
+    llm="gpt-4o-mini",
+    task_description="Answer the question using the provided context.",
+    metric=client.prompts.Metrics.CORRECTNESS,
+)
+print(f"Optimization task submitted with ID: {task_id}")
+```
+
+#### Analyze prompts
+
+Now you can explore prompts at each iteration in the UI.
+
+![Prompt Iterations](/blog/qdrant-relari/apo1.png)
+
+The optimal **System Prompt** and **Few-Shot Examples** can then be used in your RAG system to improve its performance.
+
+## Conclusion
+
+Combining **Relari** and **Qdrant** allows you to create an iterative, data-driven evaluation framework, improving your RAG system for optimal real-world performance. These methods help ensure that your application is both responsive and effective, especially when dealing with user queries or recommendations.
+
+If you’d like to get started, sign up for **free** at [Qdrant Cloud](https://cloud.qdrant.io) and [Relari](https://www.relari.ai).
+
+You can also substitute this dataset with your own by signing up for [Qdrant Cloud](https://cloud.qdrant.io) and [Relari](https://www.relari.ai).
```

Comment:
I don't see much difference in these two paragraphs. How about leaving just the second one?